### PR TITLE
Article about force HTTPS feature

### PR DESCRIPTION
**Pull request by Soulou**

### CODE REVIEW

**Comment by Soulou** [comment]
screen

File: _posts/app/2000-01-01-force-https.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+server: nginx
+date: Mon, 05 Mar 2018 10:08:15 GMT
+content-type: text/plain; charset=utf-8
+x-request-id: c87a7b84-d43f-44fb-9524-54bf666e6ff1
+strict-transport-security: max-age=31536000
+```
+
+The `strict-transport-security: max-age=31536000` ensures that browsers won't
+access the domain in an unsecure way during 1 year. By combining it with the
+HTTP to HTTPS redirection, it is now sure that users can only access the the
```
Comment (EtienneM) [inline _posts/app/2000-01-01-force-https.md]: "the the"

File: _posts/app/2000-01-01-force-https.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+application once without using an encrypted connection: at their first
+connection which is usually unauthenticated.
+
+More documentation about HSTS on [Mozilla Developer
+Network](https://developer.mozilla.org/fr/docs/S%C3%A9curit%C3%A9/HTTP_Strict_Transport_Security)
+
+## Disclaimers
+
+* Once the feature is enabled and that some users received the HSTS header,
+  they will **always** keep using HTTPS to reach your application, make sure it
```
Comment (EtienneM) [inline _posts/app/2000-01-01-force-https.md]: I would modify the order here: "reach your application, even after disabling the feature. Hence you should ensure your application works correctly through HTTPS before enabling it."

File: _posts/app/2000-01-01-force-https.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  works correctly, even after disabling the feature.
+
+* This feature adds a small lock-in to the platform as it depends of our
+  Infrastructure, if that is something you want to avoid, you can completely
```
Comment (EtienneM) [inline _posts/app/2000-01-01-force-https.md]: "Infrastructure" -> "infrastructure"
"completely" -> "easily"? Or just nothing.

File: _posts/app/2000-01-01-force-https.md [hunk 4]
```diff
@@ -64,14 +64,17 @@ Network](https://developer.mozilla.org/fr/docs/S%C3%A9curit%C3%A9/HTTP_Strict_Tr
...
-  works correctly, even after if the feature is disabled.
+* Make sure your application works correctly using HTTPS before enabling the
+  **Force HTTPS** feature. Once enabled, the application won't be reachable without HTTPS.
 
-* This feature adds a small lock-in to the platform as it depends of our
-  Infrastructure, if that is something you want to avoid, you can completely
-  implement it at your application levels. Libraries exist for most languages
-  and framework to create the exact same behavior. Here are a few examples:
+* Once the feature is enabled and that some users have received the HSTS header,
+  they will **always** keep using HTTPS to reach your application. Hance, make sure HTTPS
```
Comment (EtienneM) [inline _posts/app/2000-01-01-force-https.md]: Hance -> Hence

File: _posts/app/2000-01-01-force-https.md [hunk 5]
```diff
@@ -64,14 +64,17 @@ Network](https://developer.mozilla.org/fr/docs/S%C3%A9curit%C3%A9/HTTP_Strict_Tr
...
+  access keep working correctly, even if the feature disabled.
```
Comment (EtienneM) [inline _posts/app/2000-01-01-force-https.md]: feature disabled -> feature is disabled

File: _posts/app/2000-01-01-force-https.md [hunk 6]
```diff
@@ -64,14 +64,17 @@ Network](https://developer.mozilla.org/fr/docs/S%C3%A9curit%C3%A9/HTTP_Strict_Tr
...
+
+* This feature adds a small lock-in to the platform as it depends of Scalingo's
```
Comment (EtienneM) [inline _posts/app/2000-01-01-force-https.md]: depends of -> depends on: I think that `depends` must always be followed by `on`.

File: _posts/app/2000-01-01-force-https.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
+---
+title: "Routing setting: Force HTTPS"
```
Comment (yannski) [inline _posts/app/2000-01-01-force-https.md]: title: "Force HTTPS"

**Pull request merged**